cdpctl/validation/__init__.py

```diff
@@ -121,7 +120,9 @@ def __init__(
     @property
     def message(self):
         """Get the message."""
-        return self._template.summary.format(*self._subjects)
+        if self._subjects:
```

Comment:
do `self._template` and `self._template.summary` always exist?

Reply:
yes The template with them has to be passed as part of the Issue constructor.

Reply:
And summary is a required field for the IssueTemplate being pulled out of the issue_templates.yml